include/QuartzCore/CARenderer.h

```diff
@@ -15,12 +15,12 @@
 //******************************************************************************
 #pragma cone
 
+#import <CoreVideo/CVTime.h>
```

Comment:
Brent, you could entirely remove the @class CVTimeStamp and change the method to timeStamp:(struct CVTimeStamp*)timeStamp. 

---
In reply to: [78206753](https://github.com/Microsoft/WinObjC/pull/977#discussion_r78206753) [](ancestors = 78206753)